regex-assembly/942330.ra

```diff
@@ -3,49 +3,49 @@
 ^.?[\"'`]$
-^[\"'`\x5c]*?[\d\"'`]+\s*?and\s*?[\w\"'`][+&!@(),.-]
-^[\"'`\x5c]*?[\d\"'`]+\s*?nand\s*?[\w\"'`][+&!@(),.-]
-^[\"'`\x5c]*?[\d\"'`]+\s*?or\s*?[\w\"'`][+&!@(),.-]
-^[\"'`\x5c]*?[\d\"'`]+\s*?xor\s*?[\w\"'`][+&!@(),.-]
-^[\"'`\x5c]*?[\d\"'`]+\s*?xxor\s*?[\w\"'`][+&!@(),.-]
-^[\"'`\x5c]*?[\d\"'`]+\s*?div\s*?[\w\"'`][+&!@(),.-]
-^[\"'`\x5c]*?[\d\"'`]+\s*?like\s*?[\w\"'`][+&!@(),.-]
-^[\"'`\x5c]*?[\d\"'`]+\s*?between\s*?[\w\"'`][+&!@(),.-]
-^[\"'`\x5c]*?[\d\"'`]+\s*?not\s*?[\w\"'`][+&!@(),.-]
-^[\"'`\x5c]*?[\d\"'`]+\s*?\|\|\s*?[\w\"'`][+&!@(),.-]
-^[\"'`\x5c]*?[\d\"'`]+\s*?\&\&\s*?[\w\"'`][+&!@(),.-]
-^[\"'`\x5c]*?[^\"'`]+[\"'`]\s*?and\s*?[\w\"'`][+&!@(),.-]
-^[\"'`\x5c]*?[^\"'`]+[\"'`]\s*?nand\s*?[\w\"'`][+&!@(),.-]
-^[\"'`\x5c]*?[^\"'`]+[\"'`]\s*?or\s*?[\w\"'`][+&!@(),.-]
-^[\"'`\x5c]*?[^\"'`]+[\"'`]\s*?xor\s*?[\w\"'`][+&!@(),.-]
-^[\"'`\x5c]*?[^\"'`]+[\"'`]\s*?xxor\s*?[\w\"'`][+&!@(),.-]
-^[\"'`\x5c]*?[^\"'`]+[\"'`]\s*?div\s*?[\w\"'`][+&!@(),.-]
-^[\"'`\x5c]*?[^\"'`]+[\"'`]\s*?like\s*?[\w\"'`][+&!@(),.-]
-^[\"'`\x5c]*?[^\"'`]+[\"'`]\s*?between\s*?[\w\"'`][+&!@(),.-]
-^[\"'`\x5c]*?[^\"'`]+[\"'`]\s*?not\s*?[\w\"'`][+&!@(),.-]
-^[\"'`\x5c]*?[^\"'`]+[\"'`]\s*?\|\|\s*?[\w\"'`][+&!@(),.-]
-^[\"'`\x5c]*?[^\"'`]+[\"'`]\s*?\&\&\s*?[\w\"'`][+&!@(),.-]
+^[\"'`\x5c]*?[\d\"'`]+\s*?\band\b\s*?[\w\"'`][+&!@(),.-]
+^[\"'`\x5c]*?[\d\"'`]+\s*?\bnand\b\s*?[\w\"'`][+&!@(),.-]
+^[\"'`\x5c]*?[\d\"'`]+\s*?\bor\b\s*?[\w\"'`][+&!@(),.-]
+^[\"'`\x5c]*?[\d\"'`]+\s*?\bxor\b\s*?[\w\"'`][+&!@(),.-]
+^[\"'`\x5c]*?[\d\"'`]+\s*?\bxxor\b\s*?[\w\"'`][+&!@(),.-]
+^[\"'`\x5c]*?[\d\"'`]+\s*?\bdiv\b\s*?[\w\"'`][+&!@(),.-]
+^[\"'`\x5c]*?[\d\"'`]+\s*?\blike\b\s*?[\w\"'`][+&!@(),.-]
+^[\"'`\x5c]*?[\d\"'`]+\s*?\bbetween\b\s*?[\w\"'`][+&!@(),.-]
+^[\"'`\x5c]*?[\d\"'`]+\s*?\bnot\b\s*?[\w\"'`][+&!@(),.-]
+^[\"'`\x5c]*?[\d\"'`]+\s*?\b\|\|\b\s*?[\w\"'`][+&!@(),.-]
+^[\"'`\x5c]*?[\d\"'`]+\s*?\b\&\&\b\s*?[\w\"'`][+&!@(),.-]
+^[\"'`\x5c]*?[^\"'`]+[\"'`]\s*?\band\b\s*?[\w\"'`][+&!@(),.-]
+^[\"'`\x5c]*?[^\"'`]+[\"'`]\s*?\bnand\b\s*?[\w\"'`][+&!@(),.-]
+^[\"'`\x5c]*?[^\"'`]+[\"'`]\s*?\bor\b\s*?[\w\"'`][+&!@(),.-]
+^[\"'`\x5c]*?[^\"'`]+[\"'`]\s*?\bxor\b\s*?[\w\"'`][+&!@(),.-]
+^[\"'`\x5c]*?[^\"'`]+[\"'`]\s*?\bxxor\b\s*?[\w\"'`][+&!@(),.-]
+^[\"'`\x5c]*?[^\"'`]+[\"'`]\s*?\bdiv\b\s*?[\w\"'`][+&!@(),.-]
+^[\"'`\x5c]*?[^\"'`]+[\"'`]\s*?\blike\b\s*?[\w\"'`][+&!@(),.-]
+^[\"'`\x5c]*?[^\"'`]+[\"'`]\s*?\bbetween\b\s*?[\w\"'`][+&!@(),.-]
+^[\"'`\x5c]*?[^\"'`]+[\"'`]\s*?\bnot\b\s*?[\w\"'`][+&!@(),.-]
+^[\"'`\x5c]*?[^\"'`]+[\"'`]\s*?\b\|\|\b\s*?[\w\"'`][+&!@(),.-]
+^[\"'`\x5c]*?[^\"'`]+[\"'`]\s*?\b\&\&\b\s*?[\w\"'`][+&!@(),.-]
 [^\w\s]\w+\s*?[|-]\s*?[\"'`]\s*?\w
-@\w+\s+and\s*?[\"'`\d]+
-@\w+\s+or\s*?[\"'`\d]+
-@\w+\s+xor\s*?[\"'`\d]+
-@\w+\s+div\s*?[\"'`\d]+
-@\w+\s+like\s*?[\"'`\d]+
-@\w+\s+between\s*?[\"'`\d]+
-@[\w-]+\sand\s*?[^\w\s]
-@[\w-]+\sor\s*?[^\w\s]
-@[\w-]+\sxor\s*?[^\w\s]
-@[\w-]+\sdiv\s*?[^\w\s]
-@[\w-]+\slike\s*?[^\w\s]
-@[\w-]+\sbetween\s*?[^\w\s]
+@\w+\s+and\b\s*?[\"'`\d]+
```

Comment:
Because compared to the other, zero to unlimited spaces `\s*`, this space here is mandatory `\s+` (between one and unlimited). And the space is automatically a word boundary.

Reply:
Ah, I overlooked the `\s+` to `\s*` difference. Thanks.